cbitstech/think_feel_do_so
# Add summary tables to coach dashboard views

* Use duplicates of the Engine coach dash views
  to override tables shown in the case of a social arm
* Add methods to membership model to collect social
  interactions performed under a membership

* All participants show summaries of moods, thoughts
  etc
* Participants in social arms show summaries of social
  activity
* Add feature specs to test conditional rendering of
  tables in view
* Add methods to Membership model to aggregate likes,
  comments etc for a participant.
* Add specs to check that tables render according
  to business logic

[#89516832]
